ciml/trainer.py

```diff
@@ -555,41 +596,49 @@ def local_trainer(dataset, experiment, gpu, debug):
     training_data = gather_results.load_dataset(dataset, 'training')
     test_data = gather_results.load_dataset(dataset, 'test')
     print("Training data shape: (%d, %d)" % training_data['examples'].shape)
-    print("Evaluation data shape: (%d, %d)" % test_data['examples'].shape)
+
+    if class_label == 'node_provider':
+        label_vocabulary = set(['rax-iad', 'ovh-bhs1', 'packethost-us-west-1',
```

Comment:
This limits portability a bit. But we're not really at a place where that really matters.

Reply:
Well spotted, this was a quick fix to get things moving. I will improve on that.